stacks/openshift/application/oidc-authservice-for-appid/README.md

```diff
@@ -15,18 +16,25 @@ OIDC auth service:
 
 ## How to use
 
-This configuration should be included as part of kubeflow deployment. After a successful Kubeflow deployment, you will find no pod running in the statefulset
-`authservice` in namespace `istio-system`. It will need to fill in valid 
-configuration from the environment variables via either web console or CLI with parameters from AppID instance:
-* `OIDC_PROVIDER` - fill in the value of `oAuthServerUrl`
-* `CLIENT_ID` - fill in the value of `clientId`
-* `CLIENT_SECRET` - fill in the value of `secret`
-* `REDIRECT_URL` - fill in `https://<hostname-of-the-route-istio-ingressgateway>/login/oidc`
+Create a secret prior to kubeflow deployment by filling parameters accordingly:
+```SHELL
+kubectl create secret generic appid-application-configuration -n istio-system \
```

Comment:
Quick question, is `istio-system` namespace available before we deploy kubeflow?

Reply:
good question! no, I assume no such namespace is ready before deploying kubeflow. let me add one command